packages/perseus-editor/src/diffs/__docs__/text-diff.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+
+import TextDiff from "../text-diff";
+
+import Wrapper from "./perseus-diff-wrapper";
+
+import type {TextDiffProps} from "../text-diff";
+
+import "../../styles/perseus-editor.css";
+
+type StoryArgs = Record<string, TextDiffProps>;
+
+type Story = {
+    title: string;
+    decorators: ReadonlyArray<
+        (StoryComponent: typeof React.Component) => React.ReactElement
+    >;
+};
+
+export default {
+    title: "PerseusEditor/Diffs/Text Diff",
+    decorators: [
+        (StoryComponent) => (
+            <Wrapper>
+                <StoryComponent />
+            </Wrapper>
+        ),
+    ],
+} as Story;
+
+export const TextExample = (args: StoryArgs): React.ReactElement => {
```

Comment:
I think this can be done using Storybook's CSFv3 format... :) 
 

```suggestion
import * as React from "react";

import TextDiff from "../text-diff";

import Wrapper from "./perseus-diff-wrapper";

import type {TextDiffProps} from "../text-diff";

import "../../styles/perseus-editor.css";

import type {Meta, StoryObj} from "@storybook/react-vite";

const meta: Meta = {
    title: "PerseusEditor/Diffs/Text Diff",
    component: TextDiff,
    decorators: [
        (StoryComponent) => (
            <Wrapper>
                <StoryComponent />
            </Wrapper>
        ),
    ],
};

type Story = StoryObj<typeof ArticleRendererWithDebugUI>;

export const TextExample: Story = {
    args: {
        title: "A day in the life of a text diff",
        before: "🥱 Hello world!",
        after: "😴 Goodbye world!",
    }
}
```

Reply:
Done! Thank you!